### PR TITLE
Fix title, description and add h1 in glossary

### PR DESCRIPTION
**Pull request by RiccardoMasutti**
The `<h1>` tag was missing because we used `##` (h2) for the title. I also optimized title and description.
(SEO)

### CODE REVIEW

File: docs/glossary/Glossary-PrivacyWasabi.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
 ---
 {
   "title": "Privacy and Wasabi",
-  "description": "Explanations of common words used in Wasabi and regarding Bitcoin privacy with links to the docs for more details. This is the Wasabi documentation, an archive of knowledge about the open-source, non-custodial and privacy-focused Bitcoin wallet for desktop."
+  "description": "Explanations of common words regarding Bitcoin privacy. This is the Wasabi documentation, an archive of knowledge about the open-source, non-custodial and privacy-focused Bitcoin wallet for desktop."
```
Comment (yahiheb) [inline docs/glossary/Glossary-PrivacyWasabi.md:4]:
```suggestion
  "description": "Explanations of common words regarding Wasabi and Bitcoin privacy. This is the Wasabi documentation, an archive of knowledge about the open-source, non-custodial and privacy-focused Bitcoin wallet for desktop."
```

File: docs/glossary/Glossary-PrivacyWasabi.md [hunk 2]
```diff
@@ -1,11 +1,11 @@
 ---
 {
   "title": "Privacy and Wasabi",
```
Comment (yahiheb) [inline docs/glossary/Glossary-PrivacyWasabi.md:3]:
```suggestion
  "title": "Wasabi and Bitcoin privacy Glossary",
```

File: docs/glossary/Glossary-GeneralBitcoin.md [hunk 3]
```diff
@@ -1,11 +1,11 @@
...
 {
-  "title": "Bitcoin in general",
-  "description": "Explanations of common words used in Wasabi and regarding Bitcoin privacy with links to the docs for more details. This is the Wasabi documentation, an archive of knowledge about the open-source, non-custodial and privacy-focused Bitcoin wallet for desktop."
+  "title": "Bitcoin Glossary",
+  "description": "Explanations of common words used in Bitcoin. This is the Wasabi documentation, an archive of knowledge about the open-source, non-custodial and privacy-focused Bitcoin wallet for desktop."
 }
 ---
 
-## Bitcoin in general
+# Bitcoin Glossary
```
Comment (yahiheb) [inline docs/glossary/Glossary-GeneralBitcoin.md:8]: Why did you remove the #?

File: docs/glossary/Glossary-PrivacyWasabi.md [hunk 4]
```diff
@@ -1,11 +1,11 @@
...
-  "description": "Explanations of common words used in Wasabi and regarding Bitcoin privacy with links to the docs for more details. This is the Wasabi documentation, an archive of knowledge about the open-source, non-custodial and privacy-focused Bitcoin wallet for desktop."
+  "description": "Explanations of common words regarding Bitcoin privacy. This is the Wasabi documentation, an archive of knowledge about the open-source, non-custodial and privacy-focused Bitcoin wallet for desktop."
 }
 ---
 
-## Privacy and Wasabi
+# Privacy and Wasabi Glossary
```
Comment (yahiheb) [inline docs/glossary/Glossary-PrivacyWasabi.md:8]: Why did you remove the #?

**Review by yahiheb — CHANGES_REQUESTED** [review]

**Comment by yahiheb** [comment]
Don't open new branches in the repo.

File: docs/glossary/Glossary-PrivacyWasabi.md (hunk 4 above)
Comment (RiccardoMasutti) [reply docs/glossary/Glossary-PrivacyWasabi.md:8]: Because we need to insert an h1 tag in the page, that's usually the title.

**Review by MaxHillebrand — APPROVED** [review]
~cACK~ 

---

edit, this removes the sidebar menu - that must be fixed!

File: docs/glossary/Glossary-PrivacyWasabi.md (hunk 4 above)
Comment (yahiheb) [reply docs/glossary/Glossary-PrivacyWasabi.md:8]: Why do we need that? With this PR we don't have the sub-titles in the side menu.

**Comment by MarnixCroes** [comment]
There was no consensus here, closing.

**Pull request closed without merging**